desktop/apps/buy_art/stylesheets/buttons.styl

```diff
@@ -0,0 +1,288 @@
+// @require '../../stylus_lib'
+// @require '../../../../node_modules/artsy-ezel-components/spinners'
+// @require '../../follow_button'
+
+.circle-button
```

Comment:
Just `require` the whole file, just like we would do anywhere else :) If the file is already bundled it won't re-require. In any case, we should never be duplicating CSS anywhere.